docs/marketplace/add_to_marketplace/add_item_by_type/add_custom_resource.md

```diff
@@ -291,7 +291,13 @@ resources:
 Please note that:
 
 - the `type: custom-resource` is required to specify that this resource is a Custom Resource
-- the `version` property is included, to define this item as a version `1.0.0` with a specific release note: versions are not mandatory but highly suggested to avoid overwriting existing resources.
+- the `version` property is required and must be composed by its _name_, e.g. to define this item as a version `1.0.0`, and the _releaseNote_ to highlight the latest updates of said release
+
+:::info
+Versions are not mandatory but highly suggested to avoid overwriting existing resources.
+
+Also, it is suggested to follow the [Semantic Versioning](https://semver.org/) convention when creating new versions: this convention is used to group the different versions of the same marketplace items and determine the _latest_ version that will be used by the Console to help the user to determine the best version to use and be notified of further updates.
```

Comment:
it's actually required to provide semver compatible names. I think the "suggestion" here was meant to say: actually use semver and not call versions 3.99.99 followed by 42.1.3, etc but I guess saying that using semver is required is already providing enough information :)

```suggestion
Also, it is required to follow the [Semantic Versioning](https://semver.org/) convention when creating new versions: this convention is used to group the different versions of the same marketplace items and determine the _latest_ version that will be used by the Console to help the user to determine the best version to use and be notified of further updates.
```